org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/git/GitProvider.xtend

```diff
@@ -61,19 +67,40 @@ class GitProvider {
 	}
 
 	private def void setSshSessionFactory(CloneCommand cloneCommand) {
+		
 		val sshSessionFactory = new JschConfigSessionFactory {
 
 			override protected void configure(Host host, Session session) {
-				// do nothing
+				session.setConfig('StrictHostKeyChecking','no')
+				logger.info('''HashKnownHosts = «session.getConfig('HashKnownHosts')»''')
+				logger.info('''StrictHostKeyChecking = «session.getConfig('StrictHostKeyChecking')»''')
+			}
+			
+			// provide custom private key location (if not located at ~/.ssh/id_rsa)
+			// private custom known hosts file location (if not located at ~/.ssh/known_hosts
```

Comment:
ok, this method does both, righ? The second comment is a bit confusing. Should it be "provide" here, too, instead of "private"…?